Mathlib/Algebra/BigOperators/Order.lean

```diff
@@ -417,6 +417,17 @@ theorem prod_mono_set' (f : ι → M) : Monotone fun s ↦ ∏ x in s, f x := fu
 #align finset.prod_mono_set' Finset.prod_mono_set'
 #align finset.sum_mono_set Finset.sum_mono_set
 
+@[to_additive]
+lemma apply_le_prod_of_mem (i : ι) (hi : i ∈ s) :
```

Comment:
Arguably this should be renamed to be consistent with [`Finset.single_le_sum`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/BigOperators/Order.html#Finset.single_le_sum)

Reply:
Agreed, that seems appropriate for discoverability. I have proposed some alternative names in https://github.com/leanprover-community/mathlib4/pull/11383/commits/8207488b12655c4bfb7cacf7feb0100f06784343